spec/observables/dom/ajax-spec.ts

```diff
@@ -1124,10 +1032,365 @@ describe('ajax', () => {
       });
     });
   });
+
+  describe('with includeDownloadProgress', () => {
```

Comment:
Are there any tests that have a binary type? Not for this PR but may be something to consider adding

Reply:
Binary, as in streaming data? XHR doesn't really support that. Fetch does, though.

If you mean "binary" as in a binary data type like a `"Content-Type": "image/png"` or something... All of that is handled directly by the XHR we're using under the hood, and I'm not sure how I'd test that without an end-to-end test of some sort, which we aren't quite set up for. Right now we're mocking the XHR. I could test that within the confines of our mocks though, I guess.

Reply:
All  of that said, it should "just work", because in the event of binary data like that, the XHR in the browser will have a `responseType` of `"arraybuffer"`, and the `response` itself would be something like `Uint8Array`, which is used directly in our `AjaxResponse`, without any additional processing.